service/core.go

```diff
@@ -144,6 +149,7 @@ func (c *Core) addAPIServerRoutes() {
 	c.authhandle("/pool/{pool}/staging", handleScanStaging).Methods("GET")
 	c.authhandle("/pool/{pool}/staging/{commit}", handleCommit).Methods("POST")
 	c.authhandle("/pool/{pool}/stats", handlePoolStats).Methods("GET")
+	c.authhandle("/subscribe", handleSubscribe).Methods("GET")
```

Comment:
I think its generally better (when possible) for rest endpoints to be nouns rather than verbs. Consider calling this `/events` or `/activity`?